src/Mod/CAM/PathSimulator/AppGL/Shader.cpp

```diff
@@ -267,7 +362,292 @@ const char* FragShaderFlat =
 
     "void main()  \n"
     "{  \n"
-    "    FragColor = vec4(objectColor, 1.0);  \n"
+    "    FragColor = vec4(objectColor, 1.0); \n"
+    "}  \n";
+
+
+const char* VertShader2DFbo =
+    "#version 330 core  \n"  // ----->   add long remark for a uniform auto formatting
```

Comment:
later we need to figure better way of storing such fiels as right now the shader code is pretty much unmanageable in that form.

Reply:
You are right. but I wanted to avoid too many files to add to the release.
I will change to R(" notation - its simpler to handle.

Reply:
That is actually quite good solution!